lib/matplotlib/__init__.py

```diff
@@ -1247,20 +1247,12 @@ def _label_from_arg(y, default_name):
     return None
 
 
-_DATA_DOC_TITLE = """
-
-Notes
------
-"""
-
-_DATA_DOC_APPENDIX = """
-
-.. note::
-    In addition to the above described arguments, this function can take
-    a *data* keyword argument. If such a *data* argument is given,
+_DATA_KWARG = """\
+data : indexable object, optional
+    If given,
```

Comment:
Move that "if given" to the beginning of `replaced`, to avoid a slightly strange linebreak in the unrendered docs?

Reply:
I've moved this into the formatted message. No need to split the sentence.